### PR TITLE
Block backgrounds from style attributes

### LINKED ISSUE #1019: Block css background images in emails
Images defined in the style attribute should not be loaded unless the user asks for it (same behaviour as image tags)

### PR DESCRIPTION
**Pull request by jakobsack**
This commit fixes #1019

### CODE REVIEW

**Comment by jakobsack** [comment]
@Gomez, @ChristophWurst: any comments?

Emails with background images are sent by google plus for example

File: lib/service/htmlpurify/transformcssbackground.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+					$value);
+				return $name.':'.$value;
+			} else {
+				return $cssAttribute;
+			}
+		};
+
+		// Reassemble style
+		$cssAttributes = array_map($func, $cssAttributes);
+		$new_style = implode(';', $cssAttributes);
```
Comment (ChristophWurst) [inline lib/service/htmlpurify/transformcssbackground.php]: camel case please: ``$newStyle``

**Comment by ChristophWurst** [comment]
Code looks good. I couldn't really test it because it seems like I don't have any mail with background images :/

@jancborchardt @itheiss @aspdye could you help testing please? :)

**Comment by Gomez** [comment]
:+1: 
Works perfect! 
Tested with this code: http://backgrounds.cm/.
And the insert "HTML Option" in Thunderbird (if HTML Mail is enabled).

**Pull request merged**